dll/win32/shell32/lang/fr-FR.rc

```diff
@@ -709,6 +709,20 @@ BEGIN
     PUSHBUTTON "Modifier l'&icône...", IDC_FOLDERCUST_CHANGE_ICON, 15, 210, 75, 15
 END
 
+IDD_LINK_PROBLEM DIALOGEX 0, 0, 205, 140
```

Comment:
Do you want me to provide FR-fr translation or do you prefer a dedicated PR ?

Reply:
Translation is later...